<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="usernameRegLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordRegLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordRegCheckLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="usernameRegister">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordRegister">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordRegisterChecker">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password Check</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitReg">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
